slam-ws/src/nav_goal/bin/movebase_client

```diff
@@ -0,0 +1,21 @@
+#! /usr/bin/env python
+
+import nav_goal.movebase_client
+import rospy
+import sys
+
+if __name__ == '__main__':
+   if len(sys.argv) < 4:
+        rospy.logerr("Error: Insufficient Arguments. Usage is 'rosrun nav_goal movebase_client x y yaw' where x is distance to move in x, y is distance to move in y, and yaw is rotation in radians.")
+   else:
+	try:
+		x = float(sys.argv[1])
+		y = float(sys.argv[2])
+		yaw = float(sys.argv[3])
+
+        	rospy.init_node('movebase_client_py')
+        	result = nav_goal.movebase_client.movebase_client(x, y, yaw)
+        	if result:
+            		rospy.loginfo("Goal execution done!")
```

Comment:
Isn't this more like a `"Nav Goal sent to MoveBaseAction server!"`?
When I read `"Goal execution done!"`, it feels like a message that is printed once the robot reaches the goal - doesn't `if result` trigger once the the server receives the action, not once the robot reaches the actual goal?

Reply:
Yeah you're right, I just copied the code from HotBlack Robotics but it should be more clear, will fix